src/sdk/namespace_fs.js

```diff
@@ -682,7 +682,8 @@ class NamespaceFS {
         const fs_account_config = this.set_cur_fs_account_config(object_sdk);
         await this._load_bucket(params, fs_account_config);
 
-        if (typeof nb_native().fs.temp_file === 'function') {
+        const can_use_temp_file = Boolean(fs_account_config.backend === 'GPFS' && nb_native().fs.gpfs);
```

Comment:
fs_account_config.backend === 'GPFS'  **is not working**, how do I get the backend value that was set in nsfs.js backend: backend || (process.env.GPFS_DL_PATH ? 'GPFS' : ''),

Reply:
Oh, ok I mixed up. We take the backend only from the bucket configs, and not from the account config, which is why the previous attempt wouldn't work. We need to set the backend here:

https://github.com/noobaa/noobaa-core/blob/0a1c7746ad68621fa47b53fb53ecf6b7f9016442/src/core/nsfs.js#L126-L130

Just add another field called fs_backend like so:

```js
              fs_backend: argv.backend || (process.env.GPFS_DL_PATH ? 'GPFS' : ''),
              namespace_resource_id: undefined
        });
```

Reply:
done